### DISCUSSION TITLE
[QPROF] UI issue with get_qplan_profile

### DISCUSSION THREAD

**Discussion by yakirgb** [vertica/VerticaPy#1313] [Potential bug] [answered]
Hi, 
I'm using Version:
1.1.0-beta-1—10102024-dc6dcae061e61c0dfee1e82dac295b4d4df24f19

The UI of `get_qplan_profile` isn't clear with a large query
```
qprof_interface_1.get_qplan_profile(kind="pie")
```

With `ploty`:
![image](https://github.com/user-attachments/assets/8f324848-e303-4d5c-97e3-20c8a33bf72a)
with![image](https://github.com/user-attachments/assets/fd77d4ec-ba71-43f1-88d7-263da7d79411)
Using `highcharts`:
![image](https://github.com/user-attachments/assets/31ef1139-f50a-49f2-ad27-4039a4e47fee)
With `matplotlib`:
![image](https://github.com/user-attachments/assets/d3c797a5-4695-4d25-9cdc-702a29e56aaf)

As you can see in provided images, 
1. The path line ID  is missing/cut on pie itself on `ploty`.
2. Unable to see full path line/cost/rows on  both plotting_lib . 
3. On matplotlib that numbers missing and not possible to see all path_lines 

Also, i have similar issue with `barh`, that path line cut:
```
qprof_interface_1.get_qplan_profile(kind="barh")
```

![image](https://github.com/user-attachments/assets/4b979bcf-4950-4395-bf23-35d00a9f04ef)
![image](https://github.com/user-attachments/assets/65e6d965-2541-47be-a95b-6c628f940ee1)
![image](https://github.com/user-attachments/assets/1e97d23a-ffa7-4cac-96c9-52e6b6ac696e)

Thank you Yakir.

**Comment by yakirgb** [comment]
Also using VerticapyLab it's not working well, and path id is missing  on long queries:
![image](https://github.com/user-attachments/assets/21973797-37ce-40a5-b72d-482f2e8b5bb5)
@oualib cc

**Comment by oualib** [comment, marked as answer]
I've created an issue for this one: https://github.com/vertica/VerticaPy/issues/1328

I'm closing the discussion.
